tests/spec/path-resolution.spec.js

```diff
@@ -232,7 +233,7 @@ describe('path resolution', function() {
   });
 
   it('should properly add trailing slashes with Path.addTrailing()', function() {
-    var Path = Filer.Path;
+    let Path = Filer.Path;
```

Comment:
I think this is the same with the previous one, the path should be a const because It will not be used later on